internal/pkg/postprocessor/extractor/css.go

```diff
@@ -299,15 +355,16 @@ func parseCSS(reader io.Reader, inline bool) (links []string, atImportLinks []st
 		if gt == css.ErrorGrammar {
 			if p.Err() != nil && !errors.Is(p.Err(), io.EOF) {
 				parseErr = p.Err()
-				cssLogger.Error("error parsing CSS", "error", parseErr, "inline", inline)
+				cssLogger.Error("error parsing CSS", "inline", inline, "error", parseErr)
```

Comment:
The `parseErr` include newlines, like this:
```
unexpected token '[' in declaration on line 5 and column 5
    5:     [aria-hidden="true"] {
           ^
```

So I put the `err` parameter at the end to make the terminal log look nicer.